### PR TITLE
Blender correct out format

### PR DESCRIPTION
**Pull request by magdasta**

### CODE REVIEW

File: gnr/task/renderingtaskcollector.py [hunk 1]
```diff
@@ -163,69 +164,104 @@ def __init__(self, paste=False, width=1, height=1):
...
+            else:
+                self.darkest = d
 
-        self.accepted_exr_files.append(exr_file)
+            if self.lightest:
+                self.lightest = max(l, self.lightest)
+            else:
+                self.lightest = l
+        
+        self.accepted_exr_files.append(img_file)
```
Comment (badb) [inline gnr/task/renderingtaskcollector.py]: We should change this name if we want to accept other formats.

File: gnr/task/renderingtaskcollector.py [hunk 2]
```diff
@@ -163,69 +164,104 @@ def __init__(self, paste=False, width=1, height=1):
...
-            self.alpha_lightest = l
+            if self.alpha_lightest:
+                self.alpha_lightest = max(l, self.alpha_lightest)
+            else:
+                self.alpha_lightest = l
 
-        self.accepted_alpha_files.append(exr_file)
+        self.accepted_alpha_files.append(img_file)
 
     def finalize(self, show_progress=False):
```
Comment (badb) [inline gnr/task/renderingtaskcollector.py:201]: Can you make this function shorter? Maybe we should pack behaviour for exr in one private function and for other formats into another.

File: gnr/task/renderingtaskcollector.py [hunk 3]
```diff
@@ -282,4 +318,4 @@ def sys_test():
             print e
 
 
-    test_task_collector("test_run1", "result_64")
+    test_task_collector("test_run1", "result_64")
```
Comment (badb) [inline gnr/task/renderingtaskcollector.py]: Lets remove this test at the end of the file, it just makes a mess.

File: gnr/task/renderingtaskcollector.py [hunk 4]
```diff
@@ -163,69 +164,104 @@ def __init__(self, paste=False, width=1, height=1):
...
+            band = ""
+            for b in bands:
+                band += b
+            logger.debug("BANDS: " + band)
+            logger.debug("RES: " + str(res_x) + " " + str(res_y))
+            final_img = Image.new(band, (res_x, res_y))
+            #self.accepted_exr_files.sort()
+            offset = 0
+            for f in self.accepted_exr_files:
+                print f
```
Comment (badb) [inline gnr/task/renderingtaskcollector.py]: Is this on purpose?

File: gnr/task/renderingtaskcollector.py [hunk 5]
```diff
@@ -163,69 +164,104 @@ def __init__(self, paste=False, width=1, height=1):
...
+                if not self.paste:
+                    final_img = ImageChops.add(final_img, f)
+                else:
+                    img = Image.open(f)
+                    final_img.paste(img, (0, offset))
+                    _, img_y = img.size
+                    offset += img_y
+                    img.Image.close()
+                if show_progress:
+                    print_progress(i, len(self.accepted_exr_files))
```
Comment (badb) [inline gnr/task/renderingtaskcollector.py]: Unresolved reference 'i'

File: gnr/task/renderingtaskcollector.py (hunk 4 above)
Comment (magdasta) [inline gnr/task/renderingtaskcollector.py]: What exactly?

File: gnr/task/renderingtaskcollector.py (hunk 1 above)
Comment (magdasta) [inline gnr/task/renderingtaskcollector.py]: Sure, forgot to change it. Thanks!

File: gnr/task/renderingtaskcollector.py (hunk 4 above)
Comment (badb) [inline gnr/task/renderingtaskcollector.py]: 'print' here and above

Comment (magdasta) [inline gnr/task/renderingtaskcollector.py]: Yes, it was in the original collector as well. Remove?

Comment (badb) [inline gnr/task/renderingtaskcollector.py]: Yeah.

**Pull request merged**